metrics/web/lib/project_groups/presentation/widgets/project_group_card.dart

```diff
@@ -59,34 +59,54 @@ class _ProjectGroupCardState extends State<ProjectGroupCard> {
                     mainAxisAlignment: MainAxisAlignment.spaceBetween,
                     children: <Widget>[
                       IconLabelButton(
-                        onPressed: _showEditProjectGroupDialog,
+                        onPressed: () => _showEditProjectGroupDialog(),
```

Comment:
Maybe the previous version was better?  The same for `_showDeleteProjectGroupDialog`.